test/controllers/static_pages_controller_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
+
+class StaticPagesControllerTest < ActionController::TestCase
+  test "should get home" do
+    get :home
+    assert_response :success
+  end
+
+  test "should get help" do
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)